linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpConfig.scala

```diff
@@ -29,8 +33,9 @@ class HttpInitializer extends ProtocolInitializer.Simple {
   }
 
   protected val defaultServer = {
-    val stk = http.ErrorResponder +: Http.server.stack
+    val stk = Http.server.stack
       .replace(HttpTraceInitializer.role, HttpTraceInitializer.server)
+      .prepend(http.ErrorResponder)
```

Comment:
Yep. For instance, when an identifier cannot name a request, errors are thrown on the server stack.